server/entity/fall.go

```diff
@@ -0,0 +1,101 @@
+package entity
+
+import (
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/entity/damage"
+	"github.com/df-mc/dragonfly/server/entity/effect"
+	"github.com/df-mc/dragonfly/server/world"
+	"math"
+	"sync"
+)
+
+// FallManager handles entities that can fall.
+type FallManager struct {
+	mu           sync.Mutex
+	e            fallEntity
+	fallDistance float64
```

Comment:
This can be made an atomic.Float64 then the mutex can be removed

Reply:
done :pog: